quodlibet/quodlibet/browsers/paned/prefs.py

```diff
@@ -159,14 +183,9 @@ def __init__(self, browser):
 
         self._menu = menu = Gtk.Menu()
 
-        wide_mode = ConfigCheckMenuItem(
-            _("_Wide Mode"), "browsers", "pane_wide_mode", True)
-        wide_mode.connect("toggled", self.__wide_mode_changed, browser)
-        menu.append(wide_mode)
-
         pref_item = MenuItem(_("_Preferences"), Icons.PREFERENCES_SYSTEM)
 
-        def preferences_cb(menu_item):
+        def preferences_cb(_):
```

Comment:
Even though the parameter is not used, I think it's still a bit cleaner to make it have a descriptive name. This is nitpicking though

Reply:
Plus `_` is used throughout as an alias to `gettext`